alacritty_terminal/src/term/mod.rs

```diff
@@ -816,14 +816,17 @@ pub struct Term<T> {
     dynamic_title: bool,
 
     /// Clipboard access coupled to the active window
-    clipboard: Clipboard,
+    clipboard: Box<dyn Clipboard>,
 
     /// Proxy for sending events to the event loop
     event_proxy: T,
 
     /// Current title of the window
     title: String,
 
+    /// Title used during creation.
```

Comment:
Maybe I'm nitpicking the wrong things here, but this is slightly too vague for me. Maybe something more like:
```suggestion
    /// Default title used upon window creation.
```

---

It may be the late hour getting to me, but what is the benefit of this/why is it necessary?